docs/src/main/sphinx/release/release-357.md

```diff
@@ -0,0 +1,72 @@
+# Release 357 (XX May 2021)
+
+## General
+
+* Add support for subquery expressions that return multiple columns. E.g., `SELECT x = (VALUES (1, 'a'))`. ({issue}`7773`, {issue}`7863`)
+* Allow aggregation pushdown when `COUNT(1)` is used with `GROUP BY`. ({issue}`7251`)
+* Add support for `CURRENT_CATALOG` and `CURRENT_SCHEMA`. ({issue}`7824`)
+* Add {func}`format_number` function. ({issue}`1878`)
+* Cast `row` to `json` with field names, instead of `array`. This behavior can be restored 
+  with the `deprecated.legacy-row-to-json-cast` configuration option. ({issue}`3536`)
+* Print dynamic filters summary in `EXPLAIN ANALYZE`. ({issue}`7874)
+* Minor performance improvement for queries using `IN` predicate with short list of constants. ({issue}`7840`)
+* Reduce memory usage for queries with `WINDOW` clause. ({issue}`7947`)
+* Fix incorrect handling of row expressions in `IN` predicate, quantified comparisons and scalar subqueries. ({issue}`7797`)
```

Comment:
Maybe add "Previously ..." to explain what was incorrect, or indicate what users should look for in their queries to determine whether they should expect behavior change.